_sass/minima/fastpages-styles.scss

```diff
@@ -196,22 +196,14 @@ h6:hover .anchor-link {
   font-weight:bold;
 }
 
-// Handle Overflow With Table Output
```

Comment:
Why did you delete this comment

Reply:
because i removed everything handling overflow